src/elements/hx-switch/index.spec.js

```diff
@@ -0,0 +1,61 @@
+import { fixture, expect } from '@open-wc/testing';
```

Comment:
Please include a screenshot of your test run in the PR comments.

Reply:
Changes have been done, please review.